orx/exporter.xml

```diff
@@ -46,4 +46,31 @@
     <!-- supports texure subpath -->
     <supportsTextureSubPath>yes</supportsTextureSubPath>
 
+    <supportsPivotPoint>true</supportsPivotPoint>
+
+    <supportsAnimations>true</supportsAnimations>
+	
+	<properties>
+        <property>
+            <name>includeComments</name>
+            <type>bool</type>
+            <displayName>Include comments</displayName>
+            <toolTip>Output comments in produced result.</toolTip>
+            <isAdvanced>true</isAdvanced>
+        </property>
+        <property>
+            <name>keepInCache</name>
+            <type>bool</type>
+            <displayName>Keep in cache</displayName>
+            <toolTip>Keep textures and animations in cache.</toolTip>
+            <default>false</default>
+        </property>
+        <property>
+            <name>keyDuration</name>
+            <type>string</type>
+            <displayName>Key duration</displayName>
+            <toolTip>Animation key duration in seconds.</toolTip>
+            <default>0.15</default>
```

Comment:
How was that default of 6.66... frames/seconds chosen?

Reply:
Mostly for my own needs. Re-iterating on this, I believe we can skip this setting for now. Later on, we could parse this information from the filename.

Reply:
Changed to `0.1`. That's what you have in the tutorials 😛 . Without it, `orx` complains. So some kind of default is required. We can improve this later, using some clever logic.

Reply:
Sure, 10Hz sounds like a good default.